### ISSUE TITLE
ObjectBrowserWidget should use the last browsed location as the base

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1907]
I think the ObjectBrowserWidget should behave more like a OS "open" dialog. We should have "recent locations", it should allow people to save favorites and it should use the last browsed location as the default location.

**Comment by tiberiuichim** [comment]
Maybe even allow site integrators to declare a set of global favourite locations? I imagine each website has a set of base folders that are used frequently.

**Comment by tisto** [comment]
Global and user-specific favorites are a common request from our users. +1 from me. Might be tricky from the UX POV though.

**Comment by tiberiuichim** [comment]
My idea would be to place some tabs/icons at the bottom of the ObjectBrowser sidebar popup.

**Comment by tisto** [comment]
Yeah. That could be a good option. They are easy to reach and see at the bottom but do not interfere with the normal sidebar behavior.

**Comment by sneridagh** [comment]
+1
